website/pages/en/billing.mdx

```diff
@@ -8,7 +8,18 @@ There are two plans to use when querying subgraphs on The Graph Network.
 
 - **Free Plan**: The Free Plan includes 100,000 free monthly queries with full access to the Subgraph Studio testing environment. This plan is designed for hobbyists, hackathoners, and those with side projects to try out The Graph before scaling up.
 
-- **Growth Plan**: The Growth Plan includes everything in the Free Plan with all queries after 100,000 monthly queries requiring payments with GRT or credit card (coming soon). The Growth Plan is flexible enough to cover teams that have established dapps across a variety of use cases.
+- **Growth Plan**: The Growth Plan includes everything in the Free Plan with all queries after 100,000 monthly queries requiring payments with GRT or credit card. The Growth Plan is flexible enough to cover teams that have established dapps across a variety of use cases.
+
+<VideoEmbed youtube="Nn771WagX2I" />
+
+## Query Payments with credit card
+
+- To set up billing with Stripe, users will access Subgraph Studio (https://thegraph.com/studio/)
```

Comment:
I don't think "billing with Stripe" is the most user-friendly way to position this, this should be framed as fiat payments by credit or debit card.